docs/pages/design.js

```diff
@@ -6,10 +6,10 @@ import PageHeader from '../components/PageHeader.js';
 
 export default function DocsPage(): Node {
   return (
-    <Page title="Getting started as a designer">
-      <PageHeader name="Getting started as a designer" description="" type="guidelines" />
+    <Page title="Get started as a designer">
```

Comment:
nit: let's keep PRs focused and move unrelated changes like these out to separate cleanup PRs, especially for these larger PRs that are difficult to review already